style.css

```diff
@@ -0,0 +1,488 @@
+
+html, body {
+    width: 100%;
+}
+
+*, *::before, *::after {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    background-color: white;
+    color:#333 ;
+}
+
+a {
+     color:black;
+}
+
+a:hover {
+    color: #6C8647 ;
+}
+
+#logo {
+    font-size: 150px;
+    text-align: center;
+    font-weight: 300;
+}
+
+#sara {
+    width: 30%;
+    height: 30%;
+    float: left;
+    margin: 60px 50px 20px 0;
+    border-radius: 50px 0 50px 50px;
+    max-width: 200px;
+    max-height: 384px;
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: #333;
+    font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
+    font-weight: lighter;
+}
+
+li {
+    float: left;
```

Comment:
Float was used for positioning before positioning, flexbox and gridbox came into existance. Just for your consideration. It will still work but generally developers do not use float any more.

Reply:
I tried getting the test to wrap around the picture using Flex and Float had a nicer effect. Something to work on when I'm better with flex.